sdk/communication/Azure.Communication.PhoneNumbers/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - Added PhoneNumberAdministrationClient (originally was part of the Azure.Communication.Administration package).
 - Added support to create PhoneNumberAdministrationClient with AzureKeyCredential.
```

Comment:
The client is completely changed, please check with @minnieliu and make sure we align the changelogs